HTML/js/rejestracja.js

```diff
@@ -0,0 +1,149 @@
+window.onload = function(e){
+    console.log('onload')
+    }
+    
+    function validatereg() {
+
+        let errors = true;
+        console.log(errors)
+        if (!emailvalidation('email')) {
+            errors = false;
+        }
+        if (!requiredvalidation('lastname')) {
+            errors = false;
+        }
+        if (!requiredvalidation('firstname')) {
+            errors = false;
+        }
+        if (!requiredvalidation('username')) {
+            errors = false;
+        }
+
+        if(!birthdatevalidation('birthdate')) {
+            errors = false;
+        }
+
+        if (!checkpassword('password')) {
+            errors = false;
+        }
+
+        if (!errors ){
```

Comment:
Expected indentation of 2 spaces but found 8.
Missing space before opening brace.